src/common/tse_internal.h

```diff
@@ -34,7 +34,7 @@
  */
 
 /* NB: tse_task_private is TSE_PRIV_SIZE = 1016 bytes for now */
-#define TSE_TASK_ARG_LEN		888
+#define TSE_TASK_ARG_LEN		880
```

Comment:
This caused some issue for rebuild test. I saw it in release 0.9 test.  See daos-4416.  I thought keep it 888 should be ok?